views/index.handlebars

```diff
@@ -36,7 +36,7 @@
             <p class="i18n" data-i18n="[html]ftm-maker-request-near-me" style="margin-bottom: 0">
               Coordinate with a maker group near you. Questions? <a class="text-light" href="/faq#section-makers">Read our Maker FAQ.</a>
             </p>
-             <p class="i18n" data-i18n="[html]ftm-cta-link-back">
+            <p class="i18n" data-i18n="[html]ftm-cta-link-back">
```

Comment:
I can change these defaults, as my refactor wants to add tabs where there are handlebar references